test/index.js

```diff
@@ -24,8 +24,17 @@ const NoTranslationKeyError = require('../NoTranslationKeyError');
 
 function createFakeCompiler() {
     return {
-        plugin: bond(),
-        parser: { plugin: bond() }
+        hooks: {
```

Comment:
Apparently callbacks in these file were never triggered => tests were always green. @ViLPy and I had an idea to use memfs + use webpack's fs in this plugin to have more integral tests, than the one when everything's mocked %)